app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -67,9 +67,10 @@ def govuk_radio_button(attribute, value, *args)
 
       label_options = { value: value.to_s, class: 'govuk-label govuk-radios__label' }
       label_html = label(attribute, options[:label], label_options)
+      hint_html = hint_tag(attribute, options.merge(radio_button: true, value: value))
```

Comment:
Rather than passing in two new options, would one work?
```ruby
hint_html = hint_tag(attribute, options.merge(radio_button: value))
```

Reply:
Yes it works. I called it `radio_button_value`
And then, in the rest of the code I need to do 
`if options.key?(:radio_button_value)`
instead of 
`if options[:radio_button]`
because the value could `false`